helpt/urls.py

```diff
@@ -17,6 +17,7 @@
 from django.contrib import admin
 from rest_framework.routers import DefaultRouter
 
+from projects.adapters.github_hook import urls as github_urls
```

Comment:
That causes a circular import of Workspace -> GitHubAdater -> Workspace, thus necessitating the usage of:
`import django
Workspace = django.apps.apps.get_model('projects', 'Workspace')`

I had that first, but it seemed _unclean_ somehow. Is that still the way it should be done?